extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -496,5 +481,70 @@ private boolean isEqual(State state1, State state2) {
         }
         return colorModeIsEqual && effectIsEqual;
     }
+    
+    private boolean isReachable() {
+        try {
+            // note that InetAddress.isReachable is unreliable, see
+            // http://stackoverflow.com/questions/9922543/why-does-inetaddress-isreachable-return-false-when-i-can-ping-the-ip-address
+            // That's why we do an HTTP access instead
+
+            if (bridge != null) {
+                // If there is no connection, this line will fail
+                bridge.authenticate("invalid");
+            }
+        } catch (IOException e) {
+            return false;
+        } catch (ApiException e) {
+            if (e.getMessage().contains("SocketTimeout") || e.getMessage().contains("ConnectException")
+                    || e.getMessage().contains("NoRouteToHostException")) {
+                return false;
+            } else {
+                // this seems to be only an authentication issue
+                return true;
+            }
+        }
+        return true;
+    }
+
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        logger.debug("IP configuration check started.");
+
+        // Error handler bound to the IP address field
+        ConfigStatusInfo configStatus = new ConfigStatusInfo();
```

Comment:
the config status info object is not needed here... you can directly return e.g. an immutable list of the corresponding config status message object